src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -837,6 +854,20 @@ public function findDefinition($id)
     }
 
     /**
+     * Returns the case full id used at registration time if any.
```

Comment:
I would remove the "if any" part because we always return the id that was used when defining the definition. This makes it look like`null` might be returned in some cases too.